### ISSUE TITLE
.NET Core SDK 3.0 breaks Azure Functions SDK build

### ISSUE THREAD

**Issue by peterhuene** [dotnet/cli#10363]
## Steps to reproduce

1. Install a recent .NET Core 3.0 SDK.
2. Install the [Azure Functions core tools](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local).
3. Run `func init` and select the `dotnet`  runtime.
4. Run `func start`.

## Expected  behavior

The project should build and the Azure Functions host should start.

## Actual behavior

The build fails:

TODO: insert

## Environment data

TODO: insert

**Comment by peterhuene** [comment]
After discussion, we've decided to add back a no-op `RunResolvePublishAssemblies`, which should fix the Azure Functions SDK and unblock preview 1.

The Azure Functions SDK targets are possibly no longer needed or can be made conditional on a "2.x SDK" being used.  We will file an issue for that against the Azure Functions SDK.
